src/sheets/quadrone/actor/tabs/NpcBiographyTab.svelte

```diff
@@ -130,48 +176,18 @@
     )}
   </div>
 {/if}
-<div class="tidy-tab-row flexrow" class:hidden={editing}>
-  {#if (context.enriched.appearance !== '' && context.enriched.trait !== '' && context.enriched.ideal !== '' && context.enriched.bond !== '' && context.enriched.flaw !== '') || context.unlocked}
+<div class="tidy-tab-row flexrow" class:hidden={false}>
```

Comment:
This is meant to `hidden={editing}`

There is a single text editor, and everything else hides while it is active.